version

```diff
@@ -1 +1 @@
-1.3
\ No newline at end of file
+2.0
```

Comment:
Not sure we should be bumping up to 2.0 as this does not break binary compatibility?